a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -218,9 +218,6 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute, cascade):
         delete_LB_flow.add(a10_database_tasks.MarkVThunderStatusInDB(
             requires=a10constants.VTHUNDER,
             inject={"status": constants.PENDING_DELETE}))
-        delete_LB_flow.add(vthunder_tasks.SetupDeviceNetworkMap(
```

Comment:
We can not remove this task, rack flow need this task to get device map in vthunder.
So, it can delete unused vlan and ve interface.